### PR TITLE
New Mark Docs for Lines

### PR DESCRIPTION
**Pull request by dmadeka**
@SylvainCorlay @madridnick @ssunkara1 Here is a start at the new `bqplot` docs, I havent worked on the patches or the other subsections, but if the format works, I will do it for them too.

Let the bike-shed begin! :)

### CODE REVIEW

**Comment by dmadeka** [comment]
@rmenegaux The patches example isn't really covering a true use case to me. Any suggestions?

@madridnick @ssunkara1 @SylvainCorlay Any comments?

**Comment by SylvainCorlay** [comment]
I will look at it tomorrow.

**Comment by SylvainCorlay** [comment]
Thanks so much for opening this I will look at it tomorrow!

**Comment by rmenegaux** [comment]
Two comments:
- There should be at least one pyplot example, for example duplicating the first linechart.
- A `ColorScale` seems overkill for just 2 color values, we should add some more `y`s there

**Comment by dmadeka** [comment]
@rmenegaux  Thats fine, Im okay to do the same, buy hold stuff for N-securities and pass the array to the `color` trait

File: examples/Lines.ipynb [hunk 1]
```diff
@@ -76,47 +141,26 @@
...
-    "display(fig) "
+    "line.colors = ['DarkOrange']"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Patches"
+    "In a similar way, we can also change any attribute after the plot has been displayed to change the plot. Run each of the cells below, and try changing the attributes to explore the different features and how they effect the plot."
```
Comment (rmenegaux) [inline examples/Lines.ipynb]: @dmadeka "how they affect"

**Comment by dmadeka** [comment]
@rmenegaux All set

**Comment by dmadeka** [comment]
@SylvainCorlay Any comments?

**Comment by dmadeka** [comment]
@rmenegaux @ssunkara1 @SylvainCorlay All done. Please review and merge accordingly.

**Comment by dmadeka** [comment]
@SylvainCorlay @ssunkara1 @rmenegaux What do you guys think? I added a table and moved the examples to the appropriate folders.

**Comment by dmadeka** [comment]
@SylvainCorlay This should be good to merge

**Comment by SylvainCorlay** [comment]
:+1: All good thanks for merging

**Pull request merged**